### PR TITLE
Setup project for leaderboard

### PR DESCRIPTION
**Pull request by mandelatuks**
### in this milestone the following were done 

- Write plain HTML markup 
- Add styling for HTML
- No linters

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <div class="main-container">
+      <section class="recent-score-section">
+        <div class="recent-score">
+          <h2>Recent score</h2>
+          <button type="button" id="refresh">Refresh</button>
+        </div>
+        <div class="list-score">
+          <ul>
+            <li> name: 100</li>
+            <li> name: 20</li>
```
Comment (micheaol) [inline src/index.html:24]: Please find below the error received when I check your app in the browser:

image

Kindly fix the error.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @mandelatuks , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.html (hunk 1 above)
Comment (mandelatuks) [reply src/index.html:24]: @micheaol ,
 kindly advise me on what to do cause on my end it works properly when npm start and I have installed all the necessary loader for HTML

Comment (micheaol) [reply src/index.html:24]: @mandelatuks,

I think this docs might help : https://stackoverflow.com/questions/43494794/webpack-html-webpack-plugin-error-child-compilation-failed

Comment (mandelatuks) [reply src/index.html:24]: @micheaol 
worked on it didn't have HTML loader thank you

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @mandelatuks,

YesYouAreGIF (2)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## To Highlight
✅ No linter error
✅ Good commit messages
✅ Good PR description
✅ Project is working fine

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by mandelatuks** [comment]
@AmaduKamara 
thank you and am going to merge it now

**Pull request merged**